main.go

```diff
@@ -173,7 +173,9 @@ func origMain(isOptionSpecified bool) {
 
 	fmt.Print("exoファイルを生成中... ")
 
-	artists := fmt.Sprintf("作詞：？    作曲：%s    編曲：？\r\nVo：？   譜面作成：%s", chart.Artists, chart.Author)
+	artists_slice := strings.Split(chart.Artists, " / ")
```

Comment:
[Potato Leaves](https://github.com/sevenc-nanashi/potato_leaves) does not always have `/` in artists, so it needs fallback.

Reply:
Thanks for catching that. Is it ok now?

Reply:
Right now it checks if there's ` / ` in artists, but would it be better if it checks whether the chart comes from Chart Cyanvas or Potato Leaves?

Reply:
I changed it so that it only separates the composer and vocals if the chart is from Chart Cyanvas. Let me know what you think